src/ansys/hps/client/__version__.py

```diff
@@ -29,6 +29,10 @@
 # major, minor, patch
 __version__ = importlib_metadata.version("ansys-hps-client")
 
+__company__ = "Ansys Switzerland GmbH"
+
+__company_short__ = "Ansys"
+
```

Comment:
Please remove these, this is a pyansys package not an internal one.

Reply:
reverted changes